src/main/java/br/com/labbs/monitor/MonitorMetrics.java

```diff
@@ -81,6 +83,12 @@ public void init(boolean collectJvmMetrics, String applicationVersion, double...
                 .help("counts the size of each http response")
                 .labelNames("type", "status", "method", "addr", "isError")
                 .register(collectorRegistry);
+        
+        dependencyRequestSeconds = Histogram.build().name(DEPENDENCY_REQUESTS_SECONDS_METRIC_NAME)
+        		.help("records in a histogram the number of http requests of a dependency and their duration in seconds")
```

Comment:
@KarineValenca, please remove the `http` reference here. The protocol is generic